neural_speed/models/llama/llama.cpp

```diff
@@ -56,11 +56,15 @@ static bool llama_model_eval_internal(model_context* ctx, const model_input* inp
   const int N = inputs->n_tokens;
   const int n_past = inputs->n_past;
   const int n_total = inputs->n_total;
+
   // enforce that the first token is BOS
+#ifndef CUSTOMED_MODEL
+  // customed model does NOT have the BOS token, skip this check
   if (n_total == 0 && inputs->tokens[0] != lctx.vocab.bos_token_id) {
     fprintf(stderr, "%s: first token must be BOS\n", __func__);
     return false;
   }
+#endif
```

Comment:
temp code for debuggind?

Reply:
No. 
The customed model did not have the BOS token, while Llama model have BOS token.